test/custom-parser.test.js

```diff
@@ -1340,48 +1341,48 @@ test('contentTypeParser should add multiple custom parsers with RegExp values',
     })
   })
 
-  fastify.listen(0, err => {
-    t.error(err)
+  await fastify.listen()
```

Comment:
Instead of `.listen`, call `.ready` if it only use `.inject.`
```suggestion
  await fastify.ready()
```

Reply:
Oups totally overlooked this, good spot :p